crates/tests/src/features.rs

```diff
@@ -161,6 +161,19 @@ fn test_assert() {
     case("return_msg_sig.fe", &[], uint_token(4273672062)),
     case("return_sum_list_expression_1.fe", &[], uint_token(210)),
     case("return_sum_list_expression_2.fe", &[], uint_token(210)),
+    // unary invert
+    case("return_invert_i256.fe", &[int_token(1)], int_token(-2)),
+    case("return_invert_i128.fe", &[int_token(1)], int_token(-2)),
+    case("return_invert_i64.fe", &[int_token(4000000000)], int_token(-4000000001)),
+    case("return_invert_i32.fe", &[int_token(30000)], int_token(-30001)),
+    case("return_invert_i16.fe", &[int_token(2000)], int_token(-2001)),
+    case("return_invert_i8.fe", &[int_token(1)], int_token(-2)),
```

Comment:
Is there a special use for signed ints that I'm missing?

Since this is a bitwise operation, I think use should be restricted to unsigned ints, just like the binary bitwise operations.

Reply:
I haven't thought much about use cases but Solidity, Python, Rust (and probably other languages) all let you do this so I think we shouldn't be overly restrictive.

Solidity
```solidity
int8 y = -10;
assert(~y == 9);
```

Python:
```
>>> ~-10
9
```

Rust:
```rust
    assert_eq!(!-10i8, 9);
```